feed.py

```diff
@@ -0,0 +1,74 @@
+from flask import jsonify
+import requests
+import configparser
+
+HEADLINES = "https://newsapi.org/v2/top-headlines"
+SOURCES = "https://newsapi.org/v2/sources"
+FEED = "https://newsapi.org/v2/everything"
+
+DEFAULT_LANG="en"
+DEFAULT_SORT="popularity"
+DEFAULT_SIZE=50
+STARTING_PAGE=1
+
+class NewsAPICalls:
+    """ Wrapper class for NewsAPI calls """
+
+    def __init__(self):
+        config = configparser.ConfigParser()
+        config.read('.config/api.conf')
+        self.api_key = config['NewsAPI']['key'].strip("\'")
+
+    def get_requests(self, url: str, data=None):
+        """ helper function, send a get request to a specified url and any parameters """
```

Comment:
We should probably add some error handling to these feed functions (Some try/except logic + error logging)